buildSrc/src/main/java/com/google/firebase/gradle/plugins/semver/ClassInfo.kt

```diff
@@ -0,0 +1,99 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.gradle.plugins.semver
+
+import org.objectweb.asm.tree.ClassNode
+import org.objectweb.asm.tree.FieldNode
+import org.objectweb.asm.tree.MethodNode
+
+class ClassInfo(val _node: ClassNode, val _classNodes: Map<String, ClassNode>) {
```

Comment:
why the underscore?

also, all fields are public. Is this intended?

Reply:
yeah it was recommended online that i use _ if some other fields which are public need to be accessed. The fields are public intentionally

Reply:
From https://developer.android.com/kotlin/style-guide#backing-properties seems like the use of `_<name>` is as a backing property to allow you to expose `<name>` using a getter. But `_node` is exposed as is. And `_classNodes` are only in the call to `getAllMethods()`